doc/classes/Array.xml

```diff
@@ -6,20 +6,38 @@
 	<description>
 		Generic array which can contain several elements of any type, accessible by a numerical index starting at 0. Negative indices can be used to count from the back, like in Python (-1 is the last element, -2 the second to last, etc.).
 		[b]Example:[/b]
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var array = ["One", 2, 3, "Four"]
 		print(array[0]) # One.
 		print(array[2]) # 3.
 		print(array[-1]) # Four.
 		array[2] = "Three"
 		print(array[-2]) # Three.
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		var array = new Godot.Collections.Array{"One", 2, 3, "Four"};
+		GD.Print(array[0]); // One.
+		GD.Print(array[2]); // 3.
+		GD.Print(array[array.Count - 1]); // Four.
+		array[2] = "Three";
+		GD.Print(array[array.Count - 2]); // Three.
+		[/csharp]
+		[/codeblocks]
 		Arrays can be concatenated using the [code]+[/code] operator:
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var array1 = ["One", 2]
 		var array2 = [3, "Four"]
 		print(array1 + array2) # ["One", 2, 3, "Four"]
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		// Array concatenation is not possible with C# arrays, but is with Godot.Collections.Array.
+		var array1 = new Godot.Collections.Array("One", 2);
+		var array2 = new Godot.Collections.Array(3, "Four");
```

Comment:
Neikeq pointed out that we can also use `{` and `}`, which may be preferred...?

Reply:
Yeah, I switched to that syntax in later PRs